A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_IRSInit.js

```diff
@@ -0,0 +1,143 @@
+            IRS2Status = "IRS2 OFF"
+        }
+        let IRS3GpsString = emptyIRSGpsString
+        if (SimVar.GetSimVarValue("L:A320_Neo_ADIRS_KNOB_3", "Enum") == 1) {
+            if (checkAligned == 2) {
+                IRS3Status = "IRS3 ALIGNED ON GPS"
+            } else {
+                IRS3Status = "IRS3 ALIGNING ON GPS"
+            }
+            IRS3GpsString = IRSAlignOnPos + "[color]green"
+        } else {
+            IRS3Status = "IRS3 OFF"
+        }
+
+
+
+        mcdu.setTemplate([
+            ["IRS INIT"],
+            ["LAT" + larrowupdwn , rarrowupdwn + "LONG", "REFERENCE",],
+            ['{IrsInitFont}' + originAirportLat['deg'] + '°{IrsInitFontEnd}' + originAirportLat['min'] + '.' + Math.ceil(Number(originAirportLat['sec'] / 100)) + '[s-text]{IrsInitFont}' + originAirportLat['dir'] + "{IrsInitFontEnd} [color]blue", '{IrsInitFont}' + originAirportLon['deg'] + '°{IrsInitFontEnd}' + originAirportLon['min'] + '.' + Math.ceil(Number(originAirportLon['sec'] / 100)) + '[s-text]{IrsInitFont}' + originAirportLon['dir'] + "{IrsInitFontEnd} [color]blue", mcdu.flightPlanManager.getOrigin().ident + " [color]green"],
+            ["LAT", "LONG", "GPS POSITION"],
+            GPSPosAlign,
+            ["", "", IRS1Status],
+            ["", "", IRS1GpsString],
+            ["", "", IRS2Status],
+            ["", "", IRS2GpsString],
+            ["", "", IRS3Status],
+            ["", "", IRS3GpsString],
+            [],
+            ["<RETURN", alignMsg]
+        ])
+        
+        // IRS Font is different we loop over keywords to set correct font since at the moment we cannot adapt FMCMainDisplay.js to allow for extra keywords
+        mcdu._lineElements.forEach(function (ele) {
+            ele.forEach(function (el) {
+                if (el != null) {
+                let newHtml = el;
+                if (newHtml != null) {
+                    newHtml = newHtml.innerHTML.replace(/{IrsInitFont}/g, '<span class=\'irs-text\'>');
+                    newHtml = newHtml.replace(/{IrsInitFontEnd}/g, '</span>');
+                    el.innerHTML = newHtml;
+                }
+            }})
+        });
+
+        mcdu.onLeftInput[0] = () => {
+            lon=false
+        };
+
+        mcdu.onRightInput[0] = () => {
+            lon=true
+        };
+
+        mcdu.onLeftInput[5] = () => {
+            CDUInitPage.ShowPage1(mcdu);
+        };
+
+        mcdu.onRightInput[5] = () => {
+            if (! SimVar.GetSimVarValue("A32NX_ADIRS_PFD_ALIGNED_FIRST", "Bool") && ! SimVar.GetSimVarValue("L:A320_Neo_ADIRS_IN_ALIGN", "Bool")) {
+                SimVar.SetSimVarValue("L:A320_Neo_ADIRS_IN_ALIGN", "Bool", 1);
+            } else {
+                // Ref alignment not currently implemented
+            }
+        };
+
+
+        // This page auto-refreshes based on source material. Function will stop auto-refreshing when page has been changed.
+        autoRefresh().then()
+        async function autoRefresh() {
```

Comment:
`async`` does nothing here since the UI is entirely single-threaded.

Reply:
async isn't about multithreading. It means that the function returns a promise. This is fine.